medicai/utils/general.py

```
@@ -244,3 +245,116 @@ def interpolate_1d(input_vol, coords, axis):
+                        f"= {param.default!r}"
+                        if param.default is not inspect.Parameter.empty
+                        else ""
+                    )
+                    # annotation
+                    annot = (
+                        f": {param.annotation.__name__}"
+                        if param.annotation != inspect.Parameter.empty
+                        else ""
+                    )
+                    lines.append(f"  {pname}{annot} {default}".rstrip())
+            except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)